src/data/scripts/VayraMergedModPlugin.java

```diff
@@ -450,6 +460,25 @@ public static void setExerelinActive(String factionId, boolean active) {
         }
     }
 
+    private void makeExerelinIgnoreVayraFactions() {
+        if (EXERELIN_LOADED) {
+            /**
+             * communist_clouds
+             * science_fuckers
+             * warhawk_republic
+             * almighty_dollar
+             * ashen_keepers
+             * kadur_remnant
+             */
+            SectorManager.DO_NOT_RESPAWN_FACTIONS.add("communist_clouds");
+            SectorManager.DO_NOT_RESPAWN_FACTIONS.add("science_fuckers");
+            SectorManager.DO_NOT_RESPAWN_FACTIONS.add("warhawk_republic");
+            SectorManager.DO_NOT_RESPAWN_FACTIONS.add("almighty_dollar");
+            SectorManager.DO_NOT_RESPAWN_FACTIONS.add("ashen_keepers");
+            SectorManager.DO_NOT_RESPAWN_FACTIONS.add("kadur_remnant");
```

Comment:
consider removing kadur_remnant from here - i'm not sure if they should be exempt from respawning really. but then again, this issue with them spawning too early is also potentially problematic as well. consult with PreConceptor on this

Reply:
I think it should be removed, PreConceptor thinks so as well.
> Nah, Kadur is present at the start, it should be able to respawn if it dies and it doesn't cause any issues if its able to come back afaik. And its likely to die since its got a lot of enemies.

Reply:
Change to resemble this a bit more
```
            if (!VayraMergedModPlugin.POPULAR_FRONT_ENABLED) {
                DO_NOT_RESPAWN_FACTIONS.add("communist_clouds");
            }
            if (!VayraMergedModPlugin.COLONIAL_FACTIONS_ENABLED) {
                DO_NOT_RESPAWN_FACTIONS.add("almighty_dollar");
                DO_NOT_RESPAWN_FACTIONS.add("ashen_keepers");
                DO_NOT_RESPAWN_FACTIONS.add("science_fuckers");
                DO_NOT_RESPAWN_FACTIONS.add("warhawk_republic");
            }
```
instead of just always adding all of them, oh and also
**remove kadur_remnant from here completely**

Reply:
this shouldn't be "if they are disabled" but rather "if they are enabled". they're free to respawn otherwise, who cares.

Reply:
resolving